gtsam/hybrid/tests/testHybridFactorGraph.cpp

```diff
@@ -331,13 +331,15 @@ GaussianFactorGraph::shared_ptr batchGFG(double between,
 /* ****************************************************************************/
 // Test elimination function by eliminating x1 in *-x1-*-m1 graph.
 TEST(DCGaussianElimination, Eliminate_fully) {
-  Switching self(2);
+  Switching self(2, 1.0, 0.1);
   auto factors = self.linearizedFactorGraph;
 
   // Add measurement factors
   auto measurement_noise = noiseModel::Isotropic::Sigma(1, 0.1);
-  factors.emplace_gaussian<JacobianFactor>(X(1), I_1x1, 1 * Vector1::Ones(), measurement_noise);
-  factors.emplace_gaussian<JacobianFactor>(X(2), I_1x1, 2 * Vector1::Ones(), measurement_noise);
+  factors.emplace_gaussian<JacobianFactor>(X(1), I_1x1, 1 * Vector1::Ones(),
```

Comment:
undo this change, will cause conflict with Fan's branch (measurements added in Switching now)